test/test.js

```diff
@@ -179,37 +180,37 @@ describe('suitcss', function() {
         revert();
       });
 
-      it('should call user supplied function before linting', function(done) {
+      it('should call `beforeLint` function before linting', function(done) {
         suitcss(read('fixtures/component'), {
           root: 'test/fixtures',
           beforeLint: beforeLintStub
         }).catch(done);
 
-        expect(bemLintStub.calledOnce).to.be.ok;
-        expect(beforeLintStub.calledOnce).to.be.ok;
-        expect(beforeLintStub.calledBefore(bemLintStub)).to.be.ok;
+        expect(lintImportedFilesStub).to.be.called.once;
+        expect(beforeLintStub).to.be.called.once;
+        expect(beforeLintStub).to.have.been.calledBefore(lintImportedFilesStub);
```

Comment:
This assertion alone should cover all the cases right?
It implicitly means that `beforeLintStub` and `lintImportedFilesStub` were called.

If I am right you can remove the two lines above